osmosis/assetlist.json

```diff
@@ -12599,7 +12599,8 @@
       ]
     },
     {
-      "description": "Levana native token",
+      "description": "Levana Well-funded Perps is a protocol for perpetual swaps, which are leveraged trading contracts.
```

Comment:
Missing closing quote and comma here, then checks should pass

Reply:
```suggestion
      "description": "Levana Well-funded Perps is a protocol for perpetual swaps, which are leveraged trading contracts.",
```